wormchain/contracts/tools/__tests__/test_accountant.ts

```diff
@@ -30,7 +30,7 @@ import * as devnetConsts from "../devnet-consts.json";
 import { parseUnits } from "ethers/lib/utils";
 import { CosmWasmClient } from "@cosmjs/cosmwasm-stargate";
 
-jest.setTimeout(60000);
+jest.setTimeout(120000);
```

Comment:
Wow, this is a long running test.  Is this right?

Reply:
You will see that change in many places. That's what @evan-gray and I had to do to get tilt to pass. . .

Reply:
Yes :disappointed: adding additional load appears to have slowed down all the tests running in parallel. It may be worth revisiting if there is anything we can do to speed these up without diminishing the test surface.